client/src/App.tsx

```diff
@@ -1,34 +1,65 @@
 import { MuiThemeProvider } from '@material-ui/core';
 import { theme } from './themes/theme';
-import { BrowserRouter, Route, Redirect, Switch } from 'react-router-dom';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
 import Login from './pages/Login/Login';
 import Signup from './pages/SignUp/SignUp';
-import Dashboard from './pages/Dashboard/Dashboard';
-import { AuthProvider } from './context/useAuthContext';
+// import Dashboard from './pages/Dashboard/Dashboard';
+import TestDashboard from './pages/TestDashboard/TestDashboard';
+import Unauthorized from './pages/Unauthorized/Unauthorized';
 import { SocketProvider } from './context/useSocketContext';
 import { SnackBarProvider } from './context/useSnackbarContext';
 
+import React, { useEffect, useState } from 'react';
+import ProtectedRoute from './components/ProtectedRoute/ProtectedRoute';
+import TestLogin from './pages/TestLogin/TestLogin';
+
 import './App.css';
 
 function App(): JSX.Element {
+  const [user, setUser] = useState(false);
+
+  useEffect(() => {
+    setUser(Boolean(Number(window.localStorage.getItem('user'))) || false);
+  }, []);
+
+  useEffect(() => {
+    window.localStorage.setItem('user', JSON.stringify(Number(user)));
```

Comment:
we don't want to store the user in local storage. Authentication is already handled with the template code so this is not necessary